runtime/kusama/src/lib.rs

```diff
@@ -949,6 +949,7 @@ impl pallet_society::Config for Runtime {
 
 parameter_types! {
 	pub const MinVestedTransfer: Balance = 100 * CENTS;
```

Comment:
Do we want to reduce this to align with ED (`1 * CENTS`)

Reply:
no, we want it to be more than ED